bee-common/bee-common/src/password.rs

```diff
@@ -0,0 +1,22 @@
+// Copyright 2020-2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+//! A module that provides password utilities.
+
+use argon2::{self, Config};
+use rand::Rng;
+
+/// Generates a salt to be used for password hashing.
+pub fn generate_salt() -> [u8; 32] {
+    rand::thread_rng().gen::<[u8; 32]>()
```

Comment:
From the docs:

> ThreadRng, provided by the thread_rng function, is a handle to a thread-local CSPRNG with periodic seeding from OsRng. Because this is local, it is typically much faster than OsRng. It should be secure, though the paranoid may prefer OsRng.

How paranoid are we? `rand::rngs::OsRng::default().gen()` might be better if that answer is 'very'.

I'm pretty sure the type hint can be removed too.

Reply:
Thanks, will replace, definitely not a hot path!